test/pilosa.go

```diff
@@ -221,6 +229,13 @@ func (m *Main) RunWithTransport(host string, bindPort int, joinSeeds []string) (
 
 	m.Server.Cluster.Static = false
 
+	go func() {
+		err := m.Handler.Serve()
+		if err != nil {
+			log.Printf("Handler serve error: %v", err)
```

Comment:
We should exit the process if this error happens. Since it happens in a goroutine, we will probably want to add a channel to the Command struct, send to it in the error, and listen for it in the `select{}` in `Wait()`. We can do that after merging if you want.